e2e/autoscript-apps/README.md

```diff
@@ -0,0 +1,55 @@
+<!--toc:start-->
+
+- [Autoscript Apps](#autoscript-apps)
+- [Running](#running)
+- [Adding an application](#adding-an-application)
+<!--toc:end-->
+
+# Autoscript Apps
+
+This is a package within the repository that contains miniature applications that
+represent a unit or piece of functionality. They exist on a singular multi-page
```

Comment:
" ... represent a single feature end-to-end or a collection of similar functionalities. They exist on a single, multi-page, ... "